upgrade/install-3.0.php

```diff
@@ -79,5 +79,15 @@ function upgrade_module_3_0($object, $install = false)
 			) ENGINE='._MYSQL_ENGINE_.' DEFAULT CHARSET=utf8 AUTO_INCREMENT=1');
 	}
 
+	//Add datas to paypal order
+	Db::getInstance()->Execute('
```

Comment:
Hi @thoma202,

I see and understand your point of view, but you should not see it like this.
I have also forgot one day to implement the properly update. To solve this mistake, I had to create a new version version of the module and create an upgrade file for this NEW version.

In your case, you should have in this PR an upgrade file for the version 3.8.3.
It will allow merchants upgrading from a version prior to 3.0.0 to have this column. And if one of them has installed the module after the version 3.0.0, you can check if the column already exists to avoid duplicates.
Here is an example from cronjobs: https://github.com/PrestaShop/cronjobs/blob/master/upgrade/upgrade-1.0.6.php

Regards